docs/user/publication/index.md

```diff
@@ -0,0 +1,91 @@
+# Publish to the Graasp Library
+
+Collaboration is the cornerstone of the Graasp Library, where educators and professionals come together to share, co-create and learn. By fostering a vibrant community around a shared library of resources, Graasp enables educators to access, adapt and develop collective knowledge. This shared platform allows to focus on innovation rather than reinventing the wheel, ultimately improving students' learning experiences.
+
+![Graasp Library](graap_library.jpeg)
+
+## How Can I Publish My Content?
+
+To publish a content in the Graasp Library, open the content folder you wish to publish in the Graasp Builder and click on the publication page button.
+
+![How to open the publication page](open_publish_settings.jpeg)
+
+Now that the publication page is open, follow the rest of the documentation.
+
+:::warning
+Only folders can be published in the Graasp Library. It's also important to note that all the content inside a folder will be published too.
+:::
+
+### Publication Page
+
+The publication page allows you to edit your current item (if the item has children, you need to open the publication page for each item) and visualize how it will be rendered in the library.
```

Comment:
```suggestion
The publication page allows you to edit your current item (if the item has subcontent, you need to open the publication page for each item) and visualize how it will be rendered in the library.
```